### PR TITLE
Add multi-value support to SegmentDumpTool

### PR DESCRIPTION
**Pull request by yupeng9**
Also, add the segment dump tool as part of the pinot-tool.sh script

Currently segment dump tool supports single-value only. This change adds the multi-value support. It also makes segment dump tool a subcommand in `pinot-tool.sh` script.

### CODE REVIEW

File: pinot-tools/src/main/java/org/apache/pinot/tools/SegmentDumpTool.java [hunk 1]
```diff
@@ -91,11 +102,27 @@ public void doMain(String[] args)
...
+          int[] intArray = new int[maxNumValuesPerMVEntry];
+          int length = mItr.nextIntVal(intArray);
+          System.out.print("[");
+          for (int j = 0; j < length; j++) {
+            System.out.print(dictionaries.get(columnName).get(intArray[j]));
+            if (j != length - 1) {
+              System.out.print(",");
+            }
+          }
+          System.out.print("]");
```
Comment (fx19880617) [inline pinot-tools/src/main/java/org/apache/pinot/tools/SegmentDumpTool.java:124]: add a `System.out.print("\t");`?

Comment (yupeng9) [reply pinot-tools/src/main/java/org/apache/pinot/tools/SegmentDumpTool.java]: good catch

**Comment by yupeng9** [comment]
@mayankshriv That's true for better code reuse. Though this PR is for a minor improvement, and I believe we could the refactoring in another PR.

**Comment by mayankshriv** [comment]
> @mayankshriv That's true for better code reuse. Though this PR is for a minor improvement, and I believe we could the refactoring in another PR.

If you use PinotSegmentRecordReader, most of the code complexity you have in this PR will just go away, it will just become as simple as loop over records, and print values. I'll leave it up to you to decide.

**Comment by yupeng9** [comment]
Yup. I'll address this in another PR.
> > @mayankshriv That's true for better code reuse. Though this PR is for a minor improvement, and I believe we could the refactoring in another PR.
> 
> If you use PinotSegmentRecordReader, most of the code complexity you have in this PR will just go away, it will just become as simple as loop over records, and print values. I'll leave it up to you to decide.

**Pull request merged**